lms/djangoapps/courseware/toggles.py

```diff
@@ -61,3 +61,41 @@ def course_exit_page_is_active(course_key):
         REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(course_key) and
         COURSEWARE_MICROFRONTEND_COURSE_EXIT_PAGE.is_enabled(course_key)
     )
+
+# .. toggle_name: courseware.mfe_progress_milestones
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to display learner progress milestones in a course. Supports staged
+#    rollout to students for a new micro-frontend-based implementation of the courseware page.
+# .. toggle_use_cases: temporary, open_edx
```

Comment:
Not sure if `temporary` is correct. I guess everything is temporary in the end.  /shrug

Reply:
oh yeah I meant to say that I have no idea what these are, I just copied them from above... :shame: